db/schema.rb

```diff
@@ -11,13 +11,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160602173535) do
+ActiveRecord::Schema.define(version: 20170125055610) do
```

Comment:
Typically, you would just add another migration that modifies the existing table.  That way you don't have to manually update the timestamp.

It also creates a good record, and something to roll back to if something goes wrong.